microovn/api/services/control.go

```diff
@@ -0,0 +1,38 @@
+package services
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/canonical/lxd/lxd/response"
+	"github.com/canonical/microcluster/rest"
+	"github.com/canonical/microcluster/state"
+	"github.com/gorilla/mux"
+
+	"github.com/canonical/microovn/microovn/ovn"
+)
+
+// /1.0/services/service endpoint.
+var ServiceControlCmd = rest.Endpoint{
+	Path:   "service/{service}",
+	Delete: rest.EndpointAction{Handler: disableService, AllowUntrusted: false, ProxyTarget: true},
+	Put:    rest.EndpointAction{Handler: enableService, AllowUntrusted: false, ProxyTarget: true},
+}
+
+func enableService(s *state.State, r *http.Request) response.Response {
```

Comment:
I don't see why we would need to validate the input twice, that sounds unnecessary

Reply:
If we manage to have single source of truth for these checks I don't think it's necessarily bad to validate input in multiple places. I think it makes sense to:
* Validate in request handler to stop processing invalid request as soon as possible
* Validate in function that actually implements the enable/disable functionality because there might be multiple code paths leading to that function call.

Though if you decide to go with one validation, I agree that the second one is more important. Just make sure that the invalid can not be misused for something else in the request handler.